randovania/game_description/game_patches.py

```diff
@@ -137,91 +132,67 @@ def assign_hint(self, identifier: NodeIdentifier, hint: Hint) -> GamePatches:
 
     # Dock Connection
     def assign_dock_connections(self, assignment: Iterable[tuple[DockNode, Node]]) -> GamePatches:
-        connections = list(self.dock_connection)
-        cached_dock_connections = list(self.cached_dock_connections_from)
+        connections = self.dock_connection.copy()
 
         for source, target in assignment:
-            connections[source.node_index] = target.node_index
-            cached_dock_connections[source.node_index] = None
-            # TODO: maybe this should set the other side too?
+            connections[source.identifier] = target.identifier
```

Comment:
the todo for maybe setting the other side has been removed.
Do we not want to do that anymore?

Reply:
I'm reasonably confident the todo was about `cached_dock_connections`